### PR TITLE
Base project (hello world) created

### PR DESCRIPTION
**Pull request by OisinNolan**
Added basic config files and folders inspired by [https://github.com/Polymer/lit-html](https://github.com/Polymer/lit-html)

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "module": "es2015",
+    "lib": ["es2017", "esnext.asynciterable", "dom"],
+    //"types": ["chai", "mocha", "trusted-types"],
```
Comment (AlexLloyd0) [inline tsconfig.json:6]: Delete rather than comment if these aren't needed. We can always add them later

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
+    "module": "es2015",
+    "lib": ["es2017", "esnext.asynciterable", "dom"],
```
Comment (AlexLloyd0) [inline tsconfig.json:5]: We shouldn't need `esnext.asynciterable`. It looks like "lib" takes a default value of omitted so try leaving it out?

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/node_modules/
+
+# Compiler output
+/dist/
```
Comment (engelsdamien) [inline .gitignore:4]: add trailing newline

File: src/accessible-name.ts [hunk 4]
```diff
@@ -0,0 +1 @@
+export const greeter = (name: string) => 'Hello ' + name + '!';
```
Comment (engelsdamien) [inline src/accessible-name.ts:1]: add newline at end of file

File: tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    "noImplicitReturns": true,
+    "noFallthroughCasesInSwitch": true,
+    "noImplicitAny": true,
+    "noImplicitThis": true
+  },
+  "include": [
+    "src/**/*.ts"
+  ],
+  "exclude": []
+}
```
Comment (engelsdamien) [inline tsconfig.json:22]: newline

File: tslint.json [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+      "whitespace": [
+        true,
+        "check-branch",
+        "check-decl",
+        "check-operator",
+        "check-separator",
+        "check-type"
+      ]
+    }
+  }
```
Comment (engelsdamien) [inline tslint.json:65]: newline

**Pull request merged**